catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -130,40 +144,56 @@ private String createWkt(String[] lowerCornerPosition, String[] upperCornerPosit
         if (upperCornerPosition[0].equals(lowerCornerPosition[0]) && upperCornerPosition[1].equals(
                 lowerCornerPosition[1])) {
-            return "POINT(" + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + ")";
+            return "POINT (" + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + ")";
         }
-        return "POLYGON((" + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + ", "
+        return "POLYGON ((" + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + ", "
                 + upperCornerPosition[0] + SPACE + lowerCornerPosition[1] + ", "
                 + upperCornerPosition[0] + SPACE + upperCornerPosition[1] + ", "
                 + lowerCornerPosition[0] + SPACE + upperCornerPosition[1] + ", "
                 + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + "))";
     }
 
     /**
-     * @param coords
-     *            The latitude and longitude coordinates (in no particular order).
-     * @param axisOrder
-     *            The order that the axes are represented.
-     * @return The coordinates in LON/LAT order.
+     * @param coords    The latitude and longitude coordinates (in no particular order).
+     * @return The coordinates in an array
      */
-    private String[] getCoordinates(String coords, CswAxisOrder axisOrder) {
+    private String[] getCoordinates(String coords) {
+        return coords.split(SPACE);
+    }
 
-        switch (axisOrder) {
-        case LAT_LON: {
-            /**
-             * We want to create WKT in LON/LAT order. Since the response has the coords in LAT/LON
-             * order, we need to reverse them.
-             */
-            return StringUtils.reverseDelimited(coords, SPACE.charAt(0))
-                    .split(SPACE);
-        }
-        default: {
-            /**
-             * We want to create WKT in LON/LAT order. Since this is the order of the coords in the
-             * response, we use them as-is.
-             */
-            return coords.split(SPACE);
+    private String convertToEPSG4326(String wkt, String crs) throws CswException {
+        try {
+            Geometry geometry = wktReader.read(wkt);
+            Geometry convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry,
+                    crs);
+
+            if (convertedGeometry != null && withinEPSG4326Bounds(convertedGeometry)) {
+                return wktWriter.write(convertedGeometry);
+            } else {
+                throw new GeoFormatException();
+            }
+        } catch (ParseException | GeoFormatException e) {
+            LOGGER.debug("Unable to convert {} from {} to EPSG:4326", wkt, crs, e);
+            throw new CswException(String.format("Unable to convert %s from %s to EPSG:4326", wkt, crs));
         }
+    }
+
+    private boolean isEPSG4326(String crs) {
+        return (crs.equals("urn:x-ogc:def:crs:EPSG:6.11:4326") || crs.contains("urn:ogc:def:crs:OGC:1.3:CRS84"));
```

Comment:
It is possible the JTS Crs/CrsAuthority utilities could be used here.
Otherwise, we might simply need to look for "4326" or "CRS84" strings to identify this as a 4326 CRS.